charter.md

```diff
@@ -0,0 +1,41 @@
+# Interop Team Charter
+
+The Interop Team aims to make the web more interoperable in key areas, prioritized by web developer and user needs. This team is part of [the web-platform-tests project](https://github.com/web-platform-tests/wpt), the main test suite for the web platform.
+
+The team is responsible for maintaining [Compat 2021](https://wpt.fyi/compat2021), [Interop 2022](https://wpt.fyi/interop-2022), and defining future interoperability efforts and metrics.
+
+## Scope
+
+The team works by prioritizing specific web platform features. The minimum requirements for a feature to be in scope are:
```

Comment:
As written, it looks a bit like the interop team is setting the agenda of what should be on the Web. It probably doesn't capture the reality of browsers implementations and the industrial priorities. Basically, the interop is here to help in improving the state of the web, but not to define what's next on the Web.


```suggestion
The team works by identifying specific web platform features which lack of interoperability among browser implementations **and** are in demand by web developers. 

The minimum requirements for a feature to be in scope are:
```

Probably something to add about limited volume of work to be successful. **TODO**

Reply:
So the intention here is to say that we aim to improve interoperability. But our approach isn't to just do it broadly and generically, but to prioritize specific web platform features.

I like your wording, but wonder if the last bit can be phrased more closely to the "prioritized by web developer and user needs" at the top? We could say that it causes problems for developers or users, perhaps.

Reply:
This bit is gone now.